code/modules/mob/living/simple_animal/hostile/hostile.dm

```diff
@@ -427,6 +464,12 @@
 	else
 		return 0
 
+/mob/living/simple_animal/hostile/proc/get_vent(var/obj/machinery/atmospherics/unary/vent_pump/v)
+	//ventcrawl!
+	if(!v.welded)
+		entry_vent = v
+		Goto(get_turf(v),move_to_delay)
```

Comment:
I am deeply confused by why the Goto proc even exists
1) because it just calls another proc and no mob seems to override it
2) its name is horrendous because it would be confused with an old, highly common BASIC coding command

Reply:
And for whatever reason delay and minimum distance have a swapped order